src/views/admin/Users.vue

```diff
@@ -1,8 +1,8 @@
 <template>
```

Comment:
I think you didn't edit the right file - this is `Users.vue` - please revert the changes.